docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc

```diff
@@ -57,7 +57,7 @@ Must be a power of two.
 | -1 (override by `org.opennms.timeseries.config.ring_buffer_size`)
 
 | org.opennms.timeseries.config.writer_threads
-| Number of threads used to pull samples from the ring buffer and insert them into the time-series database.
+| Number of threads used to pull samples from the buffer and insert them into the time-series database.
```

Comment:
We should standardize how we refer to the database type. In some docs, we have "time series database," while in others (like here) we have "time-series database."

Reply:
Not hyphenated seems to be standard use, so we should not hyphenate it. Good catch!

Reply:
```suggestion
| Number of threads used to pull samples from the buffer and insert them into the time series database.
```